packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -150,7 +150,7 @@ describe('Collection aggregations tab', function () {
   let browser: CompassBrowser;
 
   before(async function () {
-    compass = await beforeTests();
+    compass = await init(this.test?.fullTitle());
```

Comment:
Since this runs in the `before`, does that mean that the `this.test?.fullTitle()` will only be the first tests name?